### PR TITLE
Forms Automated Test - LPS-131235 | master

### PR DESCRIPTION
**Pull request by ewertonluna**

### CODE REVIEW

**Comment by brittneyq** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path [hunk 1]
```diff
@@ -359,6 +359,16 @@
 	<td>//a[contains(@title,"${key_titleName}")]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>ASSERT_PARAGRAPH_TEXT_STYLE</td>
+	<td>//div[(@class='liferay-ddm-form-field-paragraph-text')]//${key_formattingStyle}[contains(text(),'Test')]</td>
```
Comment (emanuelfisilva) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path:364]: @ewertonluna Good work! Solid path.

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path [hunk 2]
```diff
@@ -359,6 +359,16 @@
 	<td>//a[contains(@title,"${key_titleName}")]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>ASSERT_PARAGRAPH_TEXT_STYLE</td>
```
Comment (emanuelfisilva) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path:363]: Hey @ewertonluna , I've changed the title of this path in order to follow the standard of other paths on this file.
Please see: https://github.com/emanuelfisilva/liferay-portal/pull/167/commits/0997789e35117592a49fe7a4574ed0a06e9227ba

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path [hunk 3]
```diff
@@ -359,6 +359,16 @@
...
+	<td>//div[(@class='liferay-ddm-form-field-paragraph-text')]//${key_formattingStyle}[contains(text(),'Test')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>IFRAME_FORMATTING_STYLES</td>
```
Comment (emanuelfisilva) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path:368]: Hey @ewertonluna , I've changed the title of this path in order to follow the standard of other paths on this file.
Please see: https://github.com/emanuelfisilva/liferay-portal/pull/167/commits/0997789e35117592a49fe7a4574ed0a06e9227ba

**Comment by emanuelfisilva** [comment]
Thanks @ewertonluna  . Great work!
Please follow: https://github.com/emanuelfisilva/liferay-portal/pull/167

**Pull request closed without merging**